### ISSUE TITLE
renameColumn will changed default from NULL to 'NULL'

### ISSUE THREAD

**Issue by kingshark8848** [doctrine/dbal#2988]
DB version:  `MariaDB 10.2.7`
Laravel version: `5.5`
doctrine/dbal version `2.6`

issue description:
When use code below to change column name,

~~~php
Schema::table('my_table', function (Blueprint $table) {
    $table->renameColumn('from', 'to');
});
~~~

it will cause error:
` "SQLSTATE[22007]: Invalid datetime format: 1292 Incorrect datetime value: 'NULL' for column 'c__x5a' at row 1 (SQL: ALTER TABLE xxx CHANGE c__x5 c__x5a DATETIME DEFAULT 'NULL')" `

the column's default value is NULL, but was changed to 'NULL' (with single quotes), which caused the error.

**Comment by belgattitude** [comment]
fixed  in master, patch will be released in 2.7

**Comment by Ocramius** [comment]
See #2824 and #2825. Closing as `duplicate`.
